### PR TITLE
Convert all wrapper module build scripts from Groovy to Kotlin

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: wrapper/testapp/build.gradle.kts [hunk 1]
```diff
@@ -15,32 +15,33 @@
 */
 
 plugins {
-    id "com.diffplug.spotless"
+    id("com.diffplug.spotless")
+    application
 }
 
-apply plugin: "java"
```
Comment (charleskorn) [inline wrapper/testapp/build.gradle.kts:21]: Is the Java plugin not required if the application plugin is applied?

Comment (sschuberth) [reply wrapper/testapp/build.gradle.kts:21]: [Exactly](https://docs.gradle.org/current/userguide/application_plugin.html):

> Applying the Application plugin also implicitly applies the Java plugin.

Comment (sschuberth) [reply wrapper/testapp/build.gradle.kts:21]: Do you want me to move that simplification out to a separate commit, mentioning the given source?

Comment (charleskorn) [reply wrapper/testapp/build.gradle.kts:21]: No, that's fine, just wanted to make sure I understood how it was magically working :)

**Comment by charleskorn** [comment]
Looks good to me, thanks for this @sschuberth!

**Pull request merged**